editor.planx.uk/src/lib/feedback.ts

```diff
@@ -95,7 +95,11 @@ export async function insertFeedbackMutation(data: {
         }
       }
     `,
-    variables: data,
+    variables: {
+      ...data,
+      feedbackScore: data.feedbackScore || null,
+      userComment: data.userComment || null,
+    },
```

Comment:
🐛 Bug! We're currently passing an empty string into this function is a user does not select a `feedbackScore`.

This can't be written to this column (nullable integer) so we end up throwing an error here and not capturing the feedback at all.

https://github.com/user-attachments/assets/7abada20-4e3d-479b-aca0-7b952107b1a3